### PR TITLE
Eibc invariant

### PR DESCRIPTION
**Pull request by lacsomot**
Closes: #108

### CODE REVIEW

File: tests/eibc_invariant_test.go [hunk 1]
```diff
@@ -0,0 +1,750 @@
...
+	numRollAppFn := 0
+	numRollAppVals := 1
+	cf := test.NewBuiltinChainFactory(zaptest.NewLogger(t), []*test.ChainSpec{
+		{
+			Name: "rollapp1",
+			ChainConfig: ibc.ChainConfig{
+				Type:                "rollapp-dym",
+				Name:                "rollapp-temp",
+				ChainID:             "rollappevm_1234-1",
+				Images:              []ibc.DockerImage{rollappEVMImage},
```
Comment (anhductn2001) [inline tests/eibc_invariant_test.go:430]: and rollapp_evm for wasm

**Review by anhductn2001 — CHANGES_REQUESTED** [review]

**Comment by trinitys7** [comment]
@lacsomot test for wasm rollapp is missing, please add  them

**Review by trinitys7 — APPROVED** [review]

File: tests/eibc_invariant_test.go [hunk 2]
```diff
@@ -0,0 +1,750 @@
...
+	})
+
+	// Start relayer
+	err = r1.StartRelayer(ctx, eRep, ibcPath)
+	require.NoError(t, err)
+	err = r2.StartRelayer(ctx, eRep, anotherIbcPath)
+	require.NoError(t, err)
+
+	epochIdentifier := json.RawMessage(fmt.Sprintf(`"%s"`, "minute"))
+	propTx, err := dymension.ParamChangeProposal(ctx, dymensionUser.KeyName(), &utils.ParamChangeProposalJSON{
```
Comment (trinitys7) [inline tests/eibc_invariant_test.go:239]: could this one be set in genesis KV for short?

Comment (lacsomot) [reply tests/eibc_invariant_test.go:239]: okay

**Review by anhductn2001 — APPROVED** [review]

**Pull request merged**